specification/redis/resource-manager/Microsoft.Cache/stable/2020-12-01/redis.json

```diff
@@ -1880,6 +1880,10 @@
           "$ref": "#/definitions/Sku",
           "description": "The SKU of the Redis cache to deploy."
         },
+        "redisVersion": {
+          "type": "string",
+          "description": "Redis version. Valid values: (4, 6)"
```

Comment:
If someone does a GET to get the current value and then they do a PUT or PATCH with a value like  "redisVersion": "4.0.14" we will accept that too right? Since it should always be a successful NO-OP to do GET/PUT with the same payload.
In that case, now I think about it, it seems like it was a bit of a pretty confusing design. But that is, in fact the way it works today, right.

Reply:
In an effort to be a little less confusing/surprising, how about if we explicitly document that only the major version number means anything in PUT requests?

Reply:
Updated description to clarify that major version is only considered in PUT/PATCH request.
PATCH is doing NO-OP, but PUT is currently throwing some errors on passing minor version and needs to be fixed.